### ISSUE TITLE
Profile Body Dimensions

### ISSUE THREAD

**Issue by Tomiha** [Rokoko/rokoko-studio-live-unity#19]
Is there an easy way to get the skeleton body dimensions of a specific profile using the Studio Live Unity plugin?

**Comment by aris-semertzidis** [comment]
Yes there is, but not every kind of body dimension. Actor full height and actor hip height is only available, because it is the only information needed to apply proper retargeting.
ActorFrame has this information and you can read it inside Actor.cs script (which is the script you attach to model), on the UpdateActor() function.

actorFrame.dimensions.totalHeight
actorFrame.dimensions.hipHeight

**Comment by Tomiha** [comment]
Thanks for the reply.
Already saw that I could access the height en hipheight, but there is no possibility to access the full profile measurements?

And a second question,
Is it possible to run the Virtual Production add-on in Rokoko Studio in VR Background mode? Rokoko only needs to access the orientation data of the tracking devices, that way the VR setup can be used for other VR apps simultaneously. (At the moment Rokoko studio takes full ownership of the VR setup, which isn't always necessary)
